### DISCUSSION TITLE
Comment edit field is missing initial trigger to set text area field height appropriately. Please fix.

### DISCUSSION THREAD

**Discussion by SetTrend** [community/community#116756] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

Currently, when I edit a comment of mine, the text area field is initially displayed using a default height. Commonly, this size is too small to see my original comment.

I need to hit any character key first for the text area field to resize appropriately.

This is awkward. Please trigger an automatic resize immediately after the text area field gets initially filled with the original content for editing.

### Guidelines

- [x] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by naveedseo110** [comment, marked as answer]
Hi there,

Thank you for sharing your experience. We apologize for the inconvenience caused by the default size of the comment editing field. We acknowledge the importance of having a comfortable [editing experience](https://kineproapk.com/). We'll prioritize implementing an automatic resize feature to address this issue promptly.

Best regards,
Naveed

**Reply by SetTrend** [reply]
Thank you so much for your excellent support!
